modules/installation/pages/uyuni-proxy-setup.adoc

```diff
@@ -0,0 +1,249 @@
+[[uyuni-proxy-setup]]
+= Uyuni Proxy Setup
+
+{productname} Proxy requires additional configuration.
+
+
+
+[[uyuni-proxy-setup-pattern]]
+== Install the [path]``suma_proxy`` pattern
+
+Either, on the server, select the [package]``patterns_uyuni_proxy`` package for installation, or make sure the [path]``uyuni_proxy`` pattern is installed using the following command on the proxy as root:
+
+----
+zypper in -t pattern uyuni_proxy
+----
+
+The new salt-broker service will be automatically started at the end of the package installation.
+This service forwards the Salt interactions to the {productname} server.
+
+// REMARK 2019-08-23, ke: Is this a Salt proxy-only feature?
```

Comment:
If this is not clear yet, it should be clarified ASAP.

Reply:
ok, I'll leave the remark in until decided.